### PR TITLE
Add new Tumbleweed logos

### PR DESCRIPTION
**Pull request by ElFrogster**
As a result of discussion in #10
Add all the different files for the new Tumbleweed logo

Not 100% sure about all the light-colored ones with the logo text

### CODE REVIEW

File: Tumbleweed/square-symbolic.svg
Comment (hellcp) [inline Tumbleweed/square-symbolic.svg:1]: Would be nice to keep the symbolics 16x16

File: Tumbleweed/favicon.ico
Comment (hellcp) [inline Tumbleweed/favicon.ico:1]: Please make it multisize like the rest of the favicons

File: Tumbleweed/light-inline.svg
Comment (hellcp) [inline Tumbleweed/light-inline.svg:1]: This is fine, though you could also make the image narrower

**Comment by hellcp** [comment]
Please use `Save As -> Optimized SVG` in inkscape and optipng too

**Comment by lkocman** [comment]
@hellcp I'm guilty of that too. Thank you for reminding us @hellcp !

**Comment by ElFrogster** [comment]
The multisize favicon and optipng things I don't know how to do.

**Comment by hellcp** [comment]
You can create multisize favicon by creating a .ico file with gimp, and creating a layer for every size. Have a look at other favicons in gimp to see how it's done there.

optipng is a command, you should be able to run it over all the pngs you are submitting

**Comment by ElFrogster** [comment]
Okay, I changed the favicon and pngs now. I think that should be all.

**Review by lkocman — COMMENTED** [review]
Only one concern from my side, and this is padding. I see that you are not using any, which generally is good. But since the rest of distros are, we could have an issue if an app would be rely on that particular icon with inconsistent padding across distros.

**Comment by ElFrogster** [comment]
Which file are you talking about specifically? I mostly tried to do things in line with what I saw elsewhere.
I removed besically all the padding on light-inline because LCP said it could be more narrow. Maybe that was overdoing it?

**Comment by lkocman** [comment]
@ElFrogster you're correct it was the inline
image


If both you and @hellcp are fine with that then I won't stand in the way, but  will do the same mod for Leap instead :-)

**Comment by lkocman** [comment]
Hello folks, any remaining blockers? I see @hellcp had some comments in the review.

**Comment by lkocman** [comment]
@ElFrogster I'm about to submit wallpaper refresh to TW so it might be a good opportunity to finish the logo refresh too.

**Comment by ElFrogster** [comment]
Is there anything that still needs to be done?

**Comment by lkocman** [comment]
@ElFrogster check comments in @hellcp's review https://github.com/openSUSE/distribution-logos/pull/12/files and mark as resolved whatever is resolved. Once we're clean we're good to go I guess.

File: Tumbleweed/square-symbolic.svg
Comment (hellcp) [reply Tumbleweed/square-symbolic.svg:1]: I would also make it edge to edge, to preserve the line width at 1

File: Tumbleweed/light-inline.svg
Comment (hellcp) [reply Tumbleweed/light-inline.svg:1]: That's a bit overboard, let's keep padding on either side of the image

**Comment by ElFrogster** [comment]
Okay, I made the changes that were asked for.

**Review by hellcp — APPROVED** [review]

**Review by lkocman — APPROVED** [review]
Fine by me. I'd prefer if @DimStar77 or @ana would merge it, since I'm not Tumbleweed Release manager. Thank you @ElFrogster !

**Comment by ana** [comment]
Looks good to me. I don't have rights to merge this PR.

**Comment by pallaswept** [comment]
(I realise this is already merged but #10 says to comment here, so...)

Community feedback:

https://old.reddit.com/r/openSUSE/comments/1gdixmb/ive_always_thought_the_dislike_towards_logo/

> Now it's...an infinity symbol with two arrows, but slightly more stylized.

It seems that outside of here, interpretations vary, even with the benefit of prior knowledge of the previous logo being an 'infinity symbol'. Maracas came up again. My favourite was "two bald people kissing" :rofl: 

Care factor?

**Pull request merged**